lib/ansible/modules/cloud/azure/azure_rm_publicipaddress_facts.py

```diff
@@ -35,6 +35,18 @@
     tags:
         description:
             - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+    format:
```

Comment:
I think finally we have decided to:
- not have format option
- document only curated format
- return raw along curated for backward compatibility
- mention in the documentation that since 2.x only documented values will be returned